kermit/src/androidMain/kotlin/co/touchlab/kermit/JvmMutableLoggerConfig.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021 Touchlab
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package co.touchlab.kermit
+
+import kotlin.jvm.Volatile
+
+internal class JvmMutableLoggerConfig : MutableLoggerConfig {
```

Comment:
should probably rename this to `AndroidMutableLoggerConfig` for clarity

Reply:
@mrf7 if a java based library or app wants to use it then wouldn't `Android` name be incorrect? `Jvm` in that sense is more generic right?

Reply:
The classes are duped in jvm and android source folders because of issues with HMPP on the jvm side. This class was previously in jvmMain and androidMain depended on it. I'd keep the name as-is for now, as we plan to fix and remove.

Reply:
(TL;DR, Gustavo and I chatted about this on slack)